### DISCUSSION TITLE
Is there anything like Sweetviz which is being regularly udpated?

### DISCUSSION THREAD

**Discussion by Connor56** [fbdesignpro/sweetviz#181] [General]
First off, I love sweetviz! I use it for the majority of my EDA, and I think it's great. But I'm slightly nervous about it being a year between updates! Is there anything else like Sweetviz that's being regularly maintained?

**Comment by fbdesignpro** [comment]
Hi @Connor56! Thanks for the kind words and "poke". FWIW I want to do an update pass in the coming weeks to get rid of the most glaring issue.

**Reply by LawAndOrder1327** [reply]
Awesome! What's the most glaring issue?

**Reply by Runormal1337** [reply]
Any update on this?

**Comment by jakob1379** [comment]
any new on this?

I would love to have sweetviz be the defacto standard after seeing an insane spike in issues at pandas profiling (now ydata-profiling)

**Comment by fbdesignpro** [comment]
FWIW again, at least we did integrate PR's fixing some long-standing issues!
